### ISSUE TITLE
Something went wrong. If this issue persists please contact us through our help center at help.openai.com.

### ISSUE THREAD

**Issue by tiro8183** [xcanwin/KeepChatGPT#24]
这个问题仍然存在

**Comment by tiro8183** [comment]
![image](https://user-images.githubusercontent.com/28704881/232947359-a17eb40f-ef75-4319-9543-287532de0d77.png)

**Comment by xcanwin** [comment]
正常来说，出现IFRNAE:BypassCF之后紧接着出现IFRAME:Expire date:xxxx，那说明已经完成了cf校验。而且我看到下方的调试窗口也正常加载了cf校验通过后的/session页面内容。所以应该不可能出现这种报错才对。
谢谢反馈，我关注一下这个问题。
另外，frame load那行后面有个select.js的文件，点进去看看是不是openai官方文件。

**Comment by PandyYang** [comment]
我也遇见了同样的问题。总是Something went wrong. If this issue persists please contact us through our help center at help.openai.com.大批人反应从4月17号开始，plus用户也开始掉线了。之前用了半个月会员，都很稳定。不知道是什么情况，估计是openai官方的封堵策略提高了。
![image](https://user-images.githubusercontent.com/30763792/232979193-a9ba1543-b5d2-4e43-a1fd-6084f0483dd1.png)

**Comment by tiro8183** [comment]
> 我也遇见了同样的问题。总是Something went wrong. If this issue persists please contact us through our help center at help.openai.com.大批人反应从4月17号开始，plus用户也开始掉线了。之前用了半个月会员，都很稳定。不知道是什么情况，估计是openai官方的封堵策略提高了。 ![image](https://user-images.githubusercontent.com/30763792/232979193-a9ba1543-b5d2-4e43-a1fd-6084f0483dd1.png)

是的, 我也是17号开始出现的, 也是plus

**Comment by tiro8183** [comment]
> 

let selectSetting, ctrlPressing = !1, altPressing = !1, frame = window.frameElement, inFrame = !1, originPos = {
    x: 5,
    y: 5
}, initScroll = {
    x: 0,
    y: 0
}, HasButtonShown = !1, translateButton = document.createElement("div");
const buttonSvg = document.createElementNS("http://www.w3.org/2000/svg", "svg")
  , buttonSvgPath = document.createElementNS("http://www.w3.org/2000/svg", "path");
let outId;
async function checkPermission() {
    return new Promise(function(t, e) {
        chrome.storage.sync.get("setting", e=>{
            t(e.setting)
        }
        )
    }
    )
}
function mouseWords(t) {
    let e, n, o, s = t.clientX, i = t.clientY;
    if (!s || !i)
        return;
    if (document.caretPositionFromPoint) {
        let t = document.caretPositionFromPoint(s, i);
        if (!t)
            return;
        e = t.offsetNode,
        n = t.offset
    } else if (document.caretRangeFromPoint) {
        let t = document.caretRangeFromPoint(s, i);
        if (!t)
            return;
        e = t.startContainer,
        n = t.startOffset
    }
    if (!e || 3 !== e.nodeType)
        return;
    let r = e.data
      , a = (o = r.slice(0, n).match(/[a-z’']+$/i)) ? o[0] : ""
      , l = (o = r.slice(n).match(/^[a-z’']+/i)) ? o[0] : "";
    if (0 === a.length && 0 === l.length)
        return;
    let c = document.createRange();
    c.setStart(e, n - a.length),
    c.setEnd(e, n + l.length);
    let u = c.getBoundingClientRect();
    if (s >= u.left && s <= u.right && i >= u.top && i <= u.bottom) {
        let t = window.getSelection();
        t.removeAllRanges(),
        t.addRange(c);
        let e = a + l;
        e && window.textRepeat !== e && mouseOverSubmit(e)
    }
    c.detach()
}
function selectInframe() {
    if (frame) {
        let t = null;
        try {
            t = window.top || window.parent
        } catch (t) {
            return
        }
        if (!t)
            return;
        inFrame = !0
    }
}
function selectTranslate(t) {
    shouldTranslate() && "off" !== selectSetting.scribble && ("direct" === selectSetting.scribble ? translateSubmit(t) : "clickIcon" === selectSetting.scribble ? showButton(t.clientX, t.clientY, t) : "translateWithCtrlPress" === selectSetting.scribble ? ctrlPressing && translateSubmit(t) : "translateWithAltPress" === selectSetting.scribble && altPressing && translateSubmit(t))
}
function mouseOverSubmit(t) {
    chrome.runtime.sendMessage({
        action: "translateSelect",
        text: t
    }, function(t) {
        disappearButton()
    })
}
function translateSubmit(t) {
    let e = window.getSelection().toString().trim()
      , n = t.clientX
      , o = t.clientY;
    if (inFrame) {
        let e = frame.getBoundingClientRect();
        n = t.clientX + e.left,
        o = t.clientY + e.top
    }
    chrome.runtime.sendMessage({
        action: "translateSelect",
        text: e,
        clientX: n,
        clientY: o
    }, function(t) {
        disappearButton()
    })
}
function buttonClickHandler(t) {
    t.preventDefault(),
    t.stopPropagation(),
    setTimeout(()=>{
        translateSubmit(t)
    }
    , 200)
}
function showButton(t, e, n) {
    "on" === selectSetting.darkMode ? (translateButton.classList.remove("tarrot_mouse_icon"),
    translateButton.classList.add("tarrot_dark_mouse_icon")) : (translateButton.classList.remove("tarrot_dark_mouse_icon"),
    translateButton.classList.add("tarrot_mouse_icon")),
    document.documentElement.appendChild(translateButton);
    let o = calculateBtnPos(t, e);
    originPos = {
        x: o.x,
        y: o.y
    },
    initScroll = {
        x: window.pageXOffset,
        y: window.pageYOffset
    },
    translateButton.style.transform = `translate(${o.x}px, ${o.y}px)`,
    translateButton.style.display = "flex",
    HasButtonShown = !0,
    outId ? (_clearTimeout(outId),
    outId = _setTimeout("trans_DeepL_reject", ()=>{
        disappearButton()
    }
    , 2e3)) : outId = _setTimeout("trans_DeepL_reject", ()=>{
        disappearButton()
    }
    , 2e3)
}
function calculateBtnPos(t, e) {
    let n = t + 5
      , o = e + 5;
    const s = document.documentElement.clientHeight
      , i = document.documentElement.clientWidth
      , r = n
      , a = o;
    return a + 22 > s && (o = e - 5 - 22),
    a < 0 && (o = e + 5),
    r + 22 > i && (n = t - 5 - 22),
    r < 0 && (n = t + 5),
    n + 22 + 5 > i && (n = i - 22 - 5),
    n < 5 && (n = 5),
    o + 22 + 5 > s && (o = s - 22 - 5),
    o < 5 && (o = 5),
    {
        x: n,
        y: o
    }
}
function shouldTranslate() {
    let t = window.getSelection()
      , e = t.toString().trim();
    return isFirefox ? e.length > 0 && !(window.isDisplayingResult && window.translateResult.originalText === e) : e.length > 0 && (t.anchorNode.nodeType === Node.TEXT_NODE || t.focusNode.nodeType === Node.TEXT_NODE) && !(window.isDisplayingResult && window.translateResult.originalText === e)
}
function detectSelect(t, e, n) {
    let o = !1;
    const s = ()=>{
        o = !0
    }
      , i = r=>{
        o ? "function" == typeof e && e(r) : "function" == typeof n && n(r),
        t.removeEventListener("mousemove", s),
        t.removeEventListener("mouseup", i)
    }
    ;
    t.addEventListener("mousemove", s),
    t.addEventListener("mouseup", i)
}
function disappearButton() {
    HasButtonShown && (document.documentElement.removeChild(translateButton),
    HasButtonShown = !1)
}
function scrollHandler() {
    if (HasButtonShown) {
        let t = initScroll.x - window.pageXOffset
          , e = initScroll.y - window.pageYOffset;
        translateButton.style.transform = `translate(${originPos.x + t}px, ${originPos.y + e}px)`
    }
}
buttonSvg.classList.add("iconsvg"),
buttonSvg.setAttribute("viewBox", "0 0 1024 1024"),
buttonSvgPath.setAttribute("d", "M549.12 643.2l-108.373333-107.093333 1.28-1.28c74.24-82.773333 127.146667-177.92 158.293333-278.613334H725.333333V170.666667H426.666667V85.333333h-85.333334v85.333334H42.666667v84.906666h476.586666A673.834667 673.834667 0 0 1 384 484.266667a672.853333 672.853333 0 0 1-98.56-142.933334h-85.333333c31.146667 69.546667 73.813333 135.253333 127.146666 194.56L110.293333 750.293333 170.666667 810.666667l213.333333-213.333334 132.693333 132.693334 32.426667-86.826667zM789.333333 426.666667h-85.333333l-192 512h85.333333l48-128h202.666667L896 938.666667h85.333333l-192-512z m-112 298.666666L746.666667 540.373333 816 725.333333h-138.666667z"),
buttonSvg.appendChild(buttonSvgPath),
translateButton.id = "tarrot_mouse_icon",
translateButton.appendChild(buttonSvg),
translateButton.addEventListener("mousedown", buttonClickHandler),
translateButton.addEventListener("contextmenu", t=>t.preventDefault()),
window.addEventListener("DOMContentLoaded", async function() {
    console.log("frame load"),
    selectInframe(),
    await checkPermission().then(t=>{
        selectSetting = t
    }
    ),
    document.addEventListener("mousedown", ()=>{
        disappearButton(),
        detectSelect(document, t=>{
            selectTranslate(t)
        }
        )
    }
    ),
    document.addEventListener("dblclick", t=>{
        selectTranslate(t)
    }
    ),
    document.addEventListener("mousemove", t=>{
        selectSetting.autoWords && "off" !== selectSetting.scribble && _setTimeout("_mouseWords", ()=>mouseWords(t), 300)
    }
    ),
    window.addEventListener("scroll", scrollHandler),
    document.addEventListener("click", t=>{
        3 === t.detail && selectTranslate(t)
    }
    ),
    document.addEventListener("keydown", t=>{
        "Control" === t.key && !ctrlPressing && (ctrlPressing = !0),
        "Alt" === t.key && !altPressing && (altPressing = !0)
    }
    ),
    document.addEventListener("keyup", t=>{
        "Control" === t.key && ctrlPressing && (ctrlPressing = !1),
        "Alt" === t.key && altPressing && (altPressing = !1)
    }
    )
}),
chrome.storage.onChanged.addListener(function(t) {
    Object.keys(t).forEach(e=>{
        let n = t[e].newValue;
        "setting" === e && (selectSetting = n)
    }
    )
});
我也不大清楚是不是官方文件

**Comment by xcanwin** [comment]
![图片](https://user-images.githubusercontent.com/14187349/232990440-562abb2b-ea38-4db1-8696-fff0753232ba.png)
单元调试了一下，是一个翻译插件，在openai进行机器人校验的时候翻译插件也自动进行了翻译，临时关掉它试试。
另外，再看看你的F12的【网络】选项卡截图，【网络】的列有很多，所以需要你拉大一点，我看看报错时你的网络请求情况。

**Comment by Mikehjx** [comment]
![NetworkError](https://user-images.githubusercontent.com/130692420/233019586-9a3791ac-08f2-4cc4-8383-d3084ece2bc8.png)
你好 出现这个问题 是因为梯子的问题吗？

**Comment by Mikehjx** [comment]
> 
问题出现的时候 正是“网络”下面 恰好没有绿色 粉色蓝色 标识的POST and GET的地方

**Comment by Mikehjx** [comment]
我也来补充出现Something was wrong……的出现问题
![Unaught](https://user-images.githubusercontent.com/130692420/233022609-11a188c5-f24f-42a7-b2d4-04f86b30bae5.png)

**Comment by Mikehjx** [comment]
上图的“网络”出现问题地方
![20230419165517](https://user-images.githubusercontent.com/130692420/233023191-c6f26932-d53d-4f10-aeea-8b656d2d6981.png)

**Comment by Mikehjx** [comment]
![AGina、](https://user-images.githubusercontent.com/130692420/233024670-23274d1f-1a98-4afd-adde-677ab25fca30.png)
2023/4/19再次出现NetWork Error

**Comment by xcanwin** [comment]
> 我也来补充出现Something was wrong……的出现问题 ![Unaught](https://user-images.githubusercontent.com/130692420/233022609-11a188c5-f24f-42a7-b2d4-04f86b30bae5.png)

看起来是你的科学ip所在地区是openai的cf机器人重点验证区域，大部分coder的科学ip不是重点验证区域，所以cf验证时会顺利通过。
建议换个ip。

**Comment by Mikehjx** [comment]
就是在那个get and post请求的时候停滞了 像被抓到了一样。待会儿回去测试一下👍

**Comment by Mikehjx** [comment]
所以说我的那个IP被重点验证的话，他不是一个时间的问题，是一种偶然性的问题吗 所以我有的时候他会报错，有的时候却不会报错> > 我也来补充出现Something was wrong……的出现问题 ![Unaught](https://user-images.githubusercontent.com/130692420/233022609-11a188c5-f24f-42a7-b2d4-04f86b30bae5.png)
> 
> 
> 
> 看起来是你的科学ip所在地区是openai的cf机器人重点验证区域，大部分coder的科学ip不是重点验证区域，所以cf验证时会顺利通过。
> 
> 建议换个ip。

**Comment by tiro8183** [comment]
> ![图片](https://user-images.githubusercontent.com/14187349/232990440-562abb2b-ea38-4db1-8696-fff0753232ba.png) 单元调试了一下，是一个翻译插件，在openai进行机器人校验的时候翻译插件也自动进行了翻译，临时关掉它试试。 另外，再看看你的F12的【网络】选项卡截图，【网络】的列有很多，所以需要你拉大一点，我看看报错时你的网络请求情况。

![image](https://user-images.githubusercontent.com/28704881/233069258-867b674e-22f3-4341-a196-3bff3129f8a4.png)
你好,这是我的网络部分截图

**Comment by Mikehjx** [comment]
> > ![图片](https://user-images.githubusercontent.com/14187349/232990440-562abb2b-ea38-4db1-8696-fff0753232ba.png) 单元调试了一下，是一个翻译插件，在openai进行机器人校验的时候翻译插件也自动进行了翻译，临时关掉它试试。 另外，再看看你的F12的【网络】选项卡截图，【网络】的列有很多，所以需要你拉大一点，我看看报错时你的网络请求情况。
> 
> ![image](https://user-images.githubusercontent.com/28704881/233069258-867b674e-22f3-4341-a196-3bff3129f8a4.png) ![image](https://user-images.githubusercontent.com/28704881/233069889-92a4ae27-0517-4024-bc4b-16d634b69377.png)
> 
> 你好,这是我的网络部分截图

为啥会拒绝访问呢 莫非是被检测出来了吗 
![Uploading 34256.png…]()

**Comment by xcanwin** [comment]
对，很可能是梯子问题，你换个国家试试。具体哪个国家的我不好推荐，因为我看到了openai所在国的网友也需要用我这个插件。
